addons/missionModules/XEH_postInit.sqf

```diff
@@ -26,3 +26,21 @@ GVAR(ehAdded) = false;
         [QGVAR(UpdateAISettings), []] call CBA_fnc_localEvent;
     };
 }] call CBA_fnc_addEventHandler;
+
+[QGVAR(banzi), {
+    params ["_group"];
+    TRACE_1("banzi eh", _group);
+
+    TRACE_4("",behaviour _unit,combatMode _group,speedMode _group,formation _group);
+    _group setBehaviour "AWARE";
+    _group setCombatMode "RED";
+    _group setSpeedMode "FULL";
+    _group setFormation "LINE";
+
+    {
+        TRACE_1("member",_x);
+        _x disableAI "AUTOCOMBAT";
+        _x disableAI "COVER";
```

Comment:
I don't want to turn them into unstoppable robots
mainly just get them up and moving
we can tweak it later, or add a 2nd module with stronger effects